portal/exercise_diet/views.py

```diff
@@ -158,31 +158,39 @@ def recipe(heading, item):
         if prev_heading_index >= 0:
             prev_heading = ordered_headings[prev_heading_index]
             prev_item_index = len(assets[prev_heading]) - 1
-            prev_asset_title = ('&larr;&nbsp;' +
-                                assets[prev_heading][prev_item_index][0])
+            prev_asset_title = ('<span class="figure">&larr;</span>' +
```

Comment:
Is there a way to move the wrapping HTML into the template?

Reply:
nice!! couldn't think of a way to remove the HTML, but this is much more readable